src/cmds/restic/cmd_ls.go

```diff
@@ -52,7 +52,7 @@ func printTree(repo *repository.Repository, id *restic.ID, prefix string) error
 	}
 
 	for _, entry := range tree.Nodes {
-		Printf(formatNode(prefix, entry, lsOptions.ListLong) + "\n")
+		Printf("%s\n", formatNode(prefix, entry, lsOptions.ListLong))
```

Comment:
Wow, I cannot believe not one of the lint, vet or other obnoxious tools let that slip. Using format strings with user supplied strings is bad.... (and yes, pointing finger at self here...)